Sources/Sentry/include/SentryUIApplication.h

```diff
@@ -41,6 +42,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSArray<UIScene *> *)getApplicationConnectedScenes:(UIApplication *)application
     API_AVAILABLE(ios(13.0), tvos(13.0));
+
+/**
+ * Return the most relevant view controller being presented in the app.
+ */
+- (nullable NSArray<UIViewController *> *)relevantViewControllers;
```

Comment:
Could you explain a bit more here? The comment speaks about a singular vc, but this returns an array.